### PR TITLE
Update Windows installation documentation

### LINKED ISSUE #504: [suggestion] Easier Installation for Windows
To install this library on Windows 10, one may follow the [tutorial](https://stable-baselines.readthedocs.io/en/master/guide/install.html#prerequisites), which instructs the user to replace the line `gym[atari,classic_control]>=0.10.9` in _setup.py_ by `gym[classic_control]>=0.10.9` before installation.

I've changed one simple detail in this _setup.py_ that will probably allow the installation on Windows 10 by simply running `python setup.py install` in prompt, without needing to ask the user to change a line of the code. At least, it worked fine for me.

Currently, the _setup.py_ code (in lines 105 to 109) is like this:

```Python
setup(name='stable_baselines',
      packages=[package for package in find_packages()
                if package.startswith('stable_baselines')],
      install_requires=[
          'gym[atari,classic_control]>=0.10.9',
```

Here is the deal. You can change this piece of code by the following one (not forgeting to `import os`):

```Python
if os.name == 'nt':
  gym_version = 'gym[classic_control]>=0.10.9'
else:
  gym_version = 'gym[atari,classic_control]>=0.10.9'

setup(name='stable_baselines',
      packages=[package for package in find_packages()
                if package.startswith('stable_baselines')],
      install_requires=[
          gym_version,
```

In this code, the change that is instructed in the tutorial is now done automatically. This is because of `os.name`, which indicates which operating system is running in your machine.

I hope this suggestion can contribute to your project at some level.

### PR DESCRIPTION
**Pull request by Miffyli**
closes #504 

Initially this started as just removing modifications needed for `atari-py` but I ended up updating the whole text: MPI is not required anymore and you can install `stable-baselines` on fresh and "standard" Python installation simply with `pip install stable-baselines` (tested myself and asked a friend to try it out). 

Also includes updates to changelog (and the missing changelog update from Tensorflow upper-bound PR).

### CODE REVIEW

File: docs/guide/install.rst [hunk 1]
```diff
@@ -39,18 +39,14 @@ Homebrew installed, run the following:
 Windows 10
 ~~~~~~~~~~
 
-We recommend using `Anaconda <https://conda.io/docs/user-guide/install/windows.html>`_ for windows users.
+For a quick start you can move straight to installing Stable-Baselines in the next step (without MPI). This supports most but not all algorithms.
 
-0. Create a new environment in the Anaconda Navigator (at least python 3.5) and install ``zlib`` in this environment.
```
Comment (araffin) [inline docs/guide/install.rst]: This part is not needed anymore? I would still keep recommending anaconda anyway, no?

Comment (Miffyli) [reply docs/guide/install.rst]: `zlib` is part of Python by default, no? If this refers to some libraries, then those are not required. We have been able to do `pip install stable-baselines` on almost fresh Python installations on Windows.

Comment (araffin) [reply docs/guide/install.rst]: > is part of Python by default, no?

no idea... but good if it is not required

Comment (Miffyli) [reply docs/guide/install.rst]: I figure it referred to this package: https://docs.python.org/3/library/zlib.html

**Pull request merged**